src/GraphQL/GraphQL.csproj

```diff
@@ -27,6 +27,7 @@
   <ItemGroup>
     <PackageReference Include="GraphQL-Parser" Version="4.1.1" />
     <PackageReference Include="Microsoft.CSharp" Version="4.5.0" />
+    <PackageReference Include="Microsoft.Extensions.DependencyInjection" Version="2.0.0" />
```

Comment:
It is very unfortunate to see such a dependency.

Reply:
You could remove it, and allow for a custom ExecutionStrategy to be passed in which creates a scope per request.  The essential bit, having the services property passed into the ResolveFieldContext from the execution, would still remain.  However, database access is always going to be scoped, so functionally I feel that it's necessary.

Reply:
> However, database access is always going to be scoped

One can argue with that. I would not say so.

Reply:
The reference has been changed to Microsoft.Extensions.DependencyInjection.Abstractions (which has no dependencies)

Reply:
This is the expected step. But the mere presence of this dependency harms.